corehq/apps/domain/management/commands/populate_messagingsettings.py

```diff
@@ -0,0 +1,37 @@
+from corehq.apps.cleanup.management.commands.populate_sql_model_from_couch_model import PopulateSQLCommand
+
+from corehq.apps.domain.models import MessagingSettings
+
+
+class Command(PopulateSQLCommand):
+    @classmethod
+    def couch_db_slug(cls):
+        return "domains"
+
+    @classmethod
+    def couch_doc_type(cls):
+        return 'Domain'
+
+    @classmethod
+    def sql_class(cls):
+        return MessagingSettings
+
+    def get_sql_obj(self, doc):
+        return self.sql_class().objects.get_or_create(domain=doc['name'])[0]
+
+    @classmethod
+    def commit_adding_migration(cls):
+        return "TODO"
+
+    @classmethod
+    def diff_couch_and_sql(cls, couch, sql):
+        couch_value = couch.get('granted_messaging_access', True)
```

Comment:
The value should default to false for new domains, but couch docs that don't have it set at all default to true, because they're legacy domains that existed before we had this attribute and they get grandfathered in. This is currently handled by `Document.wrap`: https://github.com/dimagi/commcare-hq/blob/e14fcb18fa56980f080dd50b75a8f29d9c5991c1/corehq/apps/domain/models.py#L482-L484